pkg/server/plugin/upstreamca/awssecret/awslib_fake.go

```diff
@@ -0,0 +1,61 @@
+package awssecret
```

Comment:
this file should be renamed with a `_test` suffix so that it isn't compiled as part of the normal package.